CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ include(GNUInstallDirs)
 # Configuration
 #
 option(BUILD_SHARED_LIBS "Build shared libraries (rather than static ones)" ON)
+option(HIDE_SYMBOLS "Build a libraries with hidden symbols unless they're specifically exported" ON)
```

Comment:
Please adjust as following:
- Please rename to `URIPARSER_VISIBILITY_HIDDEN`
- Please replace `a libraries` by `libraries`
- Please replace `they're` by `they are`
- Please move the new option right after existing option `URIPARSER_ENABLE_INSTALL`
- Please sync the section in the readme that is auto-listing availbale CMake options; the related shell command is documented at https://github.com/uriparser/uriparser/blame/master/README.md#L51 to make that easier

Reply:
Fixed thank you.